src/lib.rs

```diff
@@ -7,6 +7,7 @@ use std::time::Duration;
 
 pub(crate) mod dataflow;
 pub(crate) mod execution;
+pub(crate) mod input;
```

Comment:
Maybe call this Rust file / module `inputs` instead of `input` to match the resulting Python-side package?

Reply:
i was actually thinking of `inputs`-> `input` python-side, but that's more disruptive

Reply:
It's not a super big deal, but singular [`input` is a Python built-in function](https://docs.python.org/3/library/functions.html#input) and sometimes it gets funny syntax highlighting, so that's why I originally had it as plural.

Reply:
oh yep that rings a 🔔